src/main/java/io/kestra/plugin/notifications/slack/SlackTemplate.java

```diff
@@ -23,26 +23,6 @@
 @Getter
 @NoArgsConstructor
 public abstract class SlackTemplate extends SlackIncomingWebhook {
-    @Schema(
-        title = "Slack channel to send the message to"
-    )
```

Comment:
When you'll revert the removal to move to deprecation, can you add the following in the property description:

> This property works only with legacy webhook URLs, new Slack incoming webhook URLs are already tied to a specific channel.